index.html

```diff
@@ -4,24 +4,37 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>My Portfolio</title>
+    <title>Blessing's Portfolio</title>
     <link rel="stylesheet" href="styles.css">
     <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
     <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
     <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+    
 </head>
 <body>  
+    <section class="section">
     <main class="main">
         <div class="divNav">   
             <nav class="headerNav">             
                     <img class="toggleIcon" src="images/toggleIcon.png" alt="toggle">                                      
-                    <h3 class="logo">John Doe </h3> 
-                    <ul class="dskNav" >
-                        <li>Portfolio</li>
-                        <li>About</li>
-                        <li>Contact</li>
-                        <li><img src="images/Envelop.png" alt="Envelop"></li>
-                    </ul>                                       
+                    <h3 class="logo">Okpara Blessing </h3>
+                    <div id="show">
+                        <ul id="style"   >
+                            <div id="cross-icon">
+                                <img onclick="hide()" class="style-li" src="/images/close-Icon.png" alt="close-Icon">
+                              </div>
+                            <li onclick="hide()" class="style-li"><a href="#portfolioPage">Portfolio</a></li>
+                            <li onclick="hide()" class="style-li"><a href="#aboutPage">About</a></li>
+                            <li onclick="hide()" class="style-li"><a href="#contactPage">Contact</a></li>
+                            <li id="messge"><img src="images/Envelop.png" alt="Envelop"></li>
+                        </ul>                          
+                  </div> 
+                    <ul class="dskNavList">
+                        <li class="liList"><a href="#portfolioPage">Portfolio</a></li>
+                        <li class="liList"><a href="#aboutPage">About</a></li>
+                        <li class="liList"><a href="#contactPage">Contact</a></li>
+                        <li class="liList"><img src="images/Envelop.png" alt="Envelop"></li>
+                    </ul>                                        
```

Comment:
- Please make sure to follow the design guidelines as much as possible, which include:
   - [ ] Aligning the navigation links to the left of the mobile menu.
   - [ ] Separating each link with a horizontal line to match the given design.
   - [ ] Aligning the "close" button to the right of the mobile menu.
   - [ ] Using the given font family and styling to match the given design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/203623550-c198c691-0271-426c-9978-c7f67c130446.png) | ![image](https://user-images.githubusercontent.com/62290545/203623499-932b2efe-ccf0-4838-b754-bdbafecef420.png) |